support-frontend/assets/pages/weekly-subscription-checkout/helpers/deliveryDays.ts

```diff
@@ -59,14 +64,54 @@ function getTierThreeDeliveryDate(today?: number) {
 	return result;
 }
 
-function getGuardianAdLiteDate(today?: number) {
-	const firstValidDate = addDays(new Date(today ?? new Date()), 15);
-	return firstValidDate;
-}
+const productDeliveryDate = (
```

Comment:
👍  For this method and switching on the product key, nice to see the logic all located together.